sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.controller.service.ts

```diff
@@ -331,6 +331,7 @@ export class EducationProgramOfferingControllerService {
       offeringDeclaration: offering.offeringDeclaration,
       submittedDate: offering.submittedDate,
       offeringStatus: offering.offeringStatus,
+      precedingOfferingId: offering.precedingOffering?.id,
```

Comment:
Please check the references for this method and check if all reference queries are providing the `precedingOffering.id`. I believe that one of the references is not retrieving value. https://github.com/bcgov/SIMS/blob/b51eb3e15db5681886afb7d66c35855722e70d19/sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts#L286-L315

Reply:
I have changed `offering.precedingOffering.id` to `precedingOffering.id` in the `education-program-offering.service`.